modules/base/rendering/renderablemodel.cpp

```diff
@@ -144,18 +145,19 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo ModelScaleInfo = {
         "ModelScale",
         "Model Scale",
-        "This value specifies the scale for the model. If a value for the ModelScale was "
-        "provided in the asset file, you can see and change it here. If instead a unit "
-        "name was provided in the asset, this is the value that that name represents. "
-        "For example 'Centimeter' becomes 0.01. For more information see "
+        "A value that specified the scale for the model. If a numeric value was provided "
+        "in the asset file, you can see and change it here. If instead a unit name was "
+        "provided in the asset, this is the value that that name represents. For example "
+        "'Centimeter' becomes 0.01. For more information see "
         "http://wiki.openspaceproject.com/docs/builders/models/model-scale.html.",
+        // @TODO (2024-04-26, emmbr) update or remove this link? Also liks in Parameters below!
```

Comment:
This should be discussed!  Do we want links to docs pages in the Parameters and PropertyInfo descriptions?

Reply:
I think here it bites us that we reuse the descriptions between property infos and parameters structs. In general it's great to not have to write the documentation twice, but we probably want the links only in the parameter structs.  Since the users are already on the docs page to read them

Reply:
We could maybe add it in addition to the `verbatim` of the propertyinfo? 

Need to know how to specify the link in a way that is not dependent on the exact URL of the version, however.

Reply:
I agree.  We probably want to avoid maintenance of these links - seems a bit too hard to keep up to date.  And I can foresee that we might want to move pages around a bit in the future as well

So maybe remove the links then?